### PR TITLE
[luci] fix custom op export bug

### PR DESCRIPTION
**Pull request by mhs4670go**
This draft fixes bug when luci exports custom op

Related : #2222
ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

**Comment by mhs4670go** [comment]
This bug occurred because `std::unordered_map<OpCode, std::string> _custom_operator_codes;` variable. When we run `registerCustomOpcode`, its custom code(above `std::string`) was overwritten as it has the same `OpCode`(key duplicated).

**Comment by mhs4670go** [comment]
@seanshpark PTAL:)

File: compiler/luci/export/src/CircleExporterUtils.cpp [hunk 1]
```diff
@@ -90,21 +90,26 @@ namespace luci
 
 uint32_t SerializedModelData::registerBuiltinOpcode(circle::BuiltinOperator builtin_code)
 {
-  auto it = _operator_codes.find(OpCode{builtin_code});
+  auto it = _operator_codes.find(OpCode{builtin_code, ""});
```
Comment (seanshpark) [inline compiler/luci/export/src/CircleExporterUtils.cpp:93]: I think we don't need to add `, ""` for `custom_code`

File: compiler/luci/export/src/CircleExporterUtils.cpp [hunk 2]
```diff
@@ -90,21 +90,26 @@ namespace luci
...
   if (it != _operator_codes.end())
   {
     return it->second;
   }
   auto idx = static_cast<uint32_t>(_operator_codes.size());
-  _operator_codes.emplace(OpCode{builtin_code}, idx);
+  _operator_codes.emplace(OpCode{builtin_code, ""}, idx);
```
Comment (seanshpark) [inline compiler/luci/export/src/CircleExporterUtils.cpp:99]: same here

File: compiler/luci/export/src/SerializedData.h [hunk 3]
```diff
@@ -29,8 +29,19 @@ namespace luci
 struct OpCode
 {
   circle::BuiltinOperator opcode;
-
-  bool operator==(const OpCode &rhs) const { return opcode == rhs.opcode; }
+  std::string custom_code;
```
Comment (seanshpark) [inline compiler/luci/export/src/SerializedData.h:32]: (optional)
```cxx
std::string custom_code{""};
```

**Pull request merged**